UI/Selenium/Helper/Helper.cs

```diff
@@ -52,13 +53,13 @@ public static IReadOnlyCollection<IWebElement> FindMultiple(this IWebDriver driv
 
                         if (element.Count > 0 && element.ElementAt(0).Displayed && element.ElementAt(0).Enabled)
                         {
-                            Logger.Info(" The following element has been found " + element);
+                            Logger.InfoWithDate("The following element has been found " + element);
                             return element;
                         }
                     }
                     catch (NoSuchElementException e)
                     {
-                        Logger.Error(" The following error has occourred " + e);
+                        Logger.Error("The following error has occourred " + e);
```

Comment:
I need to see more detail, what element does not exist? (Element name and page name.)

Reply:
Do we need this Helper.cs? The methods in this class are not being called anywhere